pkg/ingress/tls.go

```diff
@@ -90,6 +102,7 @@ func (i *Tls) newCertNeeded() bool {
 	minimumValidity := i.ingress.KubeLego().LegoMinimumValidity()
 	timeLeft := expireTime.Sub(time.Now())
 	logger := i.Log().WithField("expire_time", expireTime)
+	certRenewalTime.WithLabelValues(strings.Join(i.Hosts(), ",")).Set(float64(expireTime.Unix()))
```

Comment:
Please make a for loop and report the expiry time per Host and not doing the join here